doc/reference/storage_powerflex.md

```diff
@@ -0,0 +1,151 @@
+  Therefore, LXD falls back to copying the volume on the local system.
+  This implicates an increased use of bandwidth due to the volumes contents being transferred over the network twice.
+
+Volume size constraints
+: In PowerFlex the size of a volume needs to be in multiples of 8 GiB.
+  This results in the smallest possible volume size of 8 GiB.
+  However, if not specified otherwise, volumes are getting thin-provisioned by LXD.
+  PowerFlex volumes can only be increased in size.
+
+Volume name size limitation
+: In PowerFlex the name of a volume cannot exceed 31 characters.
+  LXD uses certain identifiers to indicate the purpose of a volume in the storage backend.
+  See the section on [Volume names](storage-powerflex-volume-names) for more information.
+
+Sharing custom volumes between instances
+: The PowerFlex driver "simulates" volumes with content type `filesystem` by putting a file system on top of a PowerFlex volume.
+  Therefore, custom storage volumes can only be assigned to a single instance at a time.
+
+Sharing the PowerFlex storage pool between installations
+: Sharing the same PowerFlex storage pool between multiple LXD installations is not supported.
+  Due to the volume name limitations, no extra identifier can be added to distinguish between installations.
+
+## Configuration options
+
+The following configuration options are available for storage pools that use the `powerflex` driver and for storage volumes in these pools.
+
+(storage-powerflex-pool-config)=
+### Storage pool configuration
+
+Key                        | Type   | Default             | Description
+:--                        | :---   | :------             | :----------
+`powerflex.gateway`        | string | -                   | Address of the PowerFlex Gateway
+`powerflex.gateway.verify` | bool   | `true`              | Whether or not to verify the PowerFlex Gateways certificate
+`powerflex.user.name`      | string | `admin`             | User for PowerFlex Gateway authentication
+`powerflex.user.password`  | string | -                   | Password for PowerFlex Gateway authentication
+`powerflex.pool`           | string | -                   | ID of the PowerFlex storage pool. In case you want to specify the storage pool via its name, also set `powerflex.domain`
+`powerflex.domain`         | string | -                   | Name of the PowerFlex protection domain. Only required if `powerflex.pool` is specified using its name
+`powerflex.mode`           | string | the discovered mode | Gets discovered automatically if the system provides the necessary kernel modules. Currently only `nvme` is supported.
+`powerflex.sdt`            | string | one of the SDT      | PowerFlex NVMe/TCP SDT
+`powerflex.clone_copy`     | bool   | `false`             | Whether to not use PowerFlex snapshots when creating a snapshot of instances or custom volumes. See the [vTree limitations](storage-powerflex-limitations)
```

Comment:
```suggestion
`powerflex.clone_copy`     | bool   | `false`             | Whether to not use PowerFlex snapshots when creating a snapshot of instances or custom volumes (see the [vTree limitations](storage-powerflex-limitations))
```
Is it correct that if this is set to "true", we will NOT use PowerFlex snapshots?

Reply:
Yes, when you set `powerflex.clone_copy=true` on the storage pool the driver will create  cloned copies instead of snapshots when creating LXD volume snapshots.

Reply:
The explanation feels a bit backwards - but let's fix this when we have more space than a field in a table. :)